src/components/Checkbox.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View, Pressable, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Icon from './Icon';
+import {Checkmark} from './Icon/Expensicons';
+
+const propTypes = {
+    // Whether checkbox is checked
+    isChecked: PropTypes.bool.isRequired,
+
+    // A function that is called when the box/label is clicked on
+    onClick: PropTypes.func.isRequired,
+
+    // Text that appears next to check box
+    label: PropTypes.string,
+};
+
+const defaultProps = {
+    label: '',
+};
+
+const Checkbox = ({
+    isChecked, onClick, label,
+}) => (
+    <View style={styles.flexRow}>
+        <Pressable onPress={() => onClick(!isChecked)}>
+            <View style={[styles.checkboxContainer, isChecked && styles.checkedContainer]}>
+                <Icon src={Checkmark} fill="white" height={14} width={14} />
+            </View>
+        </Pressable>
+        {label ? (
+            <Pressable onPress={() => onClick(!isChecked)}>
+                <Text style={[styles.ml2, styles.textP]}>
+                    {label}
+                </Text>
+            </Pressable>
+        ) : null}
```

Comment:
This can be simplified: 
```suggestion
        {label && (
            <Pressable onPress={() => onClick(!isChecked)}>
                <Text style={[styles.ml2, styles.textP]}>
                    {label}
                </Text>
            </Pressable>
        )}
```